app/src/main/java/com/duckduckgo/app/settings/SettingsActivity.kt

```diff
@@ -291,20 +291,26 @@ class SettingsActivity : DuckDuckGoActivity() {
     private fun updateNetPSettings(networkProtectionState: Boolean, networkProtectionWaitlistState: NetPWaitlistState) {
         with(viewsPrivacy) {
             when (networkProtectionWaitlistState) {
-                NetPWaitlistState.InBeta -> {
-                    netpPSetting.show()
-                    when (networkProtectionState) {
-                        true -> R.string.netpSettingsConnected
-                        false -> R.string.netpSettingsDisconnected
-                    }.run {
-                        netpPSetting.setSecondaryText(getString(this))
-                    }
-                    val netPItemStatus = if (networkProtectionState) {
-                        CheckListItem.CheckItemStatus.ENABLED
+                is NetPWaitlistState.InBeta -> {
+                    if (networkProtectionWaitlistState.termsAccepted) {
+                        netpPSetting.show()
+                        when (networkProtectionState) {
+                            true -> R.string.netpSettingsConnected
+                            false -> R.string.netpSettingsDisconnected
+                        }.run {
+                            netpPSetting.setSecondaryText(getString(this))
+                        }
+                        val netPItemStatus = if (networkProtectionState) {
+                            CheckListItem.CheckItemStatus.ENABLED
+                        } else {
+                            CheckListItem.CheckItemStatus.WARNING
+                        }
+                        netpPSetting.setItemStatus(netPItemStatus)
                     } else {
-                        CheckListItem.CheckItemStatus.WARNING
+                        netpPSetting.show()
+                        netpPSetting.setSecondaryText(getString(R.string.netpSettingsNeverEnabled))
+                        netpPSetting.setItemStatus(CheckListItem.CheckItemStatus.DISABLED)
```

Comment:
This was feedback from [here](https://app.asana.com/0/0/1205230811122378/1205255564162048/f)